### PR TITLE
Added more feature and feature for policy history Feature-627

### PR DESCRIPTION
**Pull request by jaind1046**
Added more feature and steps in policy history Feature-627

### CODE REVIEW

File: specs/features/content-management-policy-editing.feature [hunk 1]
```diff
@@ -13,24 +13,29 @@ Feature: Content Management Policy Editing
 
 
     @TEST-188
-    Scenario: A user can cancel any updates that they have done to the policy by pressing cancel
-        When user has updated sections of the current policy
+    Scenario Outline: A user can cancel any updates that they have done to the policy by pressing cancel
+        When I change one of the <ContentFlags> for required file types <fileType> to <FlagType>
         When the user presses the Cancel button
```
Comment (NadHodeGW) [inline specs/features/content-management-policy-editing.feature:18]: Please use I instead of "the user"

**Review by NadHodeGW — CHANGES_REQUESTED** [review]
see comment

File: specs/features/content-management-policy-editing.feature (hunk 1 above)
Comment (jaind1046) [reply specs/features/content-management-policy-editing.feature]: Done. Replaced all. Will take care of this in the future also. Thanks.

**Comment by jaind1046** [comment]
@NadHodeGW It has all Scenario of issues-20 except TODO in the code with the upcoming PR with POLICY history test cases.

**Review by NadHodeGW — CHANGES_REQUESTED** [review]
Please add any required data in the corresponding scenario as example tables rather than putting them in the step's file

**Comment by jaind1046** [comment]
> Please add any required data in the corresponding scenario as example tables rather than putting them in the step's file

Hi Nadege.
These are data As User is giving "Embedded Files" in his own language in feature file .
But at the application page object we are putting field "embeddedFiles" as it is map with key as non space string. So there should be a mapping between those.
If you want me to remove these mapping than in featurte file every where these variables should be in camel case in which case we use to put variable in code.

**Comment by jaind1046** [comment]
> Please add any required data in the corresponding scenario as example tables rather than putting them in the step's file

@NadHodeGW Updated as per your comments also added the test case for policy history.

**Review by NadHodeGW — APPROVED** [review]

**Pull request merged**